pkg/images/images_test.go

```diff
@@ -47,100 +47,100 @@ func Test_toKeep(t *testing.T) {
 			wantErr: false,
 		},
 		{
-			name: "2.7.0 < deprecated < 2.8.0 ",
+			name: "2.7.0 < deprecated < 2.9.0 ",
 			args: args{info: types.K8sVersionInfo{
 				MinRancherVersion:       "",
 				MaxRancherVersion:       "",
-				DeprecateRancherVersion: "2.7.2",
+				DeprecateRancherVersion: "2.8.2",
 			}},
 			want:    true,
 			wantErr: false,
 		},
 		{
-			name: "2.7.0 < deprecated < 2.8.0 - 2",
+			name: "2.7.0 < deprecated < 2.9.0 - 2",
 			args: args{info: types.K8sVersionInfo{
 				MinRancherVersion:       "",
 				MaxRancherVersion:       "",
-				DeprecateRancherVersion: "2.7.5-alpha1",
+				DeprecateRancherVersion: "2.8.5-alpha1",
 			}},
 			want:    true,
 			wantErr: false,
 		},
 		{
-			name: "deprecated = 2.8.0",
+			name: "deprecated = 2.9.0",
 			args: args{info: types.K8sVersionInfo{
 				MinRancherVersion:       "",
 				MaxRancherVersion:       "",
-				DeprecateRancherVersion: "2.8.0",
+				DeprecateRancherVersion: "2.9.0",
 			}},
 			want:    true,
 			wantErr: false,
 		},
 		{
-			name: "min < 2.7.0 < 2.8.0 < max",
+			name: "min < 2.7.0 < 2.9.0 < max",
 			args: args{info: types.K8sVersionInfo{
 				MinRancherVersion:       "2.6.12-alpha2",
-				MaxRancherVersion:       "2.8.2",
+				MaxRancherVersion:       "2.9.2",
 				DeprecateRancherVersion: "",
 			}},
 			want:    true,
 			wantErr: false,
 		},
 		{
-			name: "min < 2.7.0 < max = 2.8.0",
+			name: "min < 2.7.0 < max = 2.9.0",
 			args: args{info: types.K8sVersionInfo{
 				MinRancherVersion:       "2.6.12-alpha2",
-				MaxRancherVersion:       "2.8.0",
+				MaxRancherVersion:       "2.9.0",
 				DeprecateRancherVersion: "",
 			}},
 			want:    true,
 			wantErr: false,
 		},
 		{
-			name: "min < 2.7.0 < max < 2.8.0",
+			name: "min < 2.7.0 < max < 2.9.0",
 			args: args{info: types.K8sVersionInfo{
 				MinRancherVersion:       "2.6.12-alpha2",
-				MaxRancherVersion:       "2.7.12-patch1",
+				MaxRancherVersion:       "2.8.12-patch1",
 				DeprecateRancherVersion: "",
 			}},
 			want:    true,
 			wantErr: false,
 		},
 		{
-			name: "min = 2.7.0 < max < 2.8.0",
+			name: "min = 2.7.0 < max < 2.9.0",
 			args: args{info: types.K8sVersionInfo{
 				MinRancherVersion:       "2.7.0",
-				MaxRancherVersion:       "2.7.5",
+				MaxRancherVersion:       "2.8.5",
 				DeprecateRancherVersion: "",
 			}},
 			want:    true,
 			wantErr: false,
 		},
 		{
-			name: "min = 2.7.0 < max = 2.8.0",
+			name: "min = 2.7.0 < max = 2.9.0",
 			args: args{info: types.K8sVersionInfo{
 				MinRancherVersion:       "2.7.0",
-				MaxRancherVersion:       "2.8.0",
+				MaxRancherVersion:       "2.9.0",
 				DeprecateRancherVersion: "",
 			}},
 			want:    true,
 			wantErr: false,
 		},
 		{
-			name: "2.7.0 < min < max < 2.8.0",
+			name: "2.7.0 < min < max < 2.9.0",
```

Comment:
The following (new) test case will fail: 
```
		{
			name: "min = 2.7.2 < max = 2.7.9",
			args: args{release: map[string]interface{}{
				"minChannelServerVersion": "v2.7.2",
				"maxChannelServerVersion": "v2.7.9",
				"version":                 "v1.20.15+rke2r2"},
			},
			want:    false,
			wantErr: false,
		},
```

In the file `images.go`,  the boundaries need to be 2.8.0 and 2.9.0

Reply:
@jiaqiluo Could you explain why the boundaries need to be 2.8.0 and 2.9.0? We still want the 2.7 versions to load in KDM right for upgrade scenarios?

Reply:
@jiaqiluo I forgot, looks like that isn't related to the generate logic. @chiukapoor saw all the 2.7 entries getting removed from regsync.yaml when updating to 2.8.0 so I asked to set it to 2.7.0. Are we okay with those entries being removed?

Reply:
@kinarashah, it is a good question. I think we should keep all tags for completeness, so the regsync file in the dev-v2.9 branch contains all tags of all images used in rancher v2.7.x and v2.8.x. 
This comment can be resolved if we all agree that.

Reply:
@jiaqiluo Ya, I had the same thought process. So we'll keep the changes as is for now. Thanks!